<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchResultDetailedInfoU</class>
 <widget class="QWidget" name="SearchResultDetailedInfoU">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*基础背景与字体*/
/*background&amp;font*/

QWidget
{
	background-color: hsl(210, 50%, 98%);
	font: 10pt &quot;Microsoft YaHei UI&quot;;
}

/*通用普通按钮*/
/*Normal Push Button*/

QPushButton,QToolButton
{
	color: hsl(0, 0%, 10%);
	border-style: solid;
	border-width: 1px;
	border-radius: 3px;
	border-color: hsl(210, 5%, 90%);
	background-color: hsl(210, 5%, 100%);
	padding: 4px 10px 4px 10px;
}
QPushButton:hover,QToolButton:hover
{
	color: hsl(0, 0%, 5%);
	border-color: hsl(210, 10%, 88%);
	background-color: hsl(210, 10%, 98%);
}
QPushButton:pressed,QToolButton:pressed
{
	color: hsl(0, 0%, 0%);
	border-color: hsl(210, 15%, 85%);
	background-color: hsl(210, 15%, 95%);
}
QPushButton:disabled,QToolButton:disabled
{
	color: hsl(210, 0%, 60%);
	border-width: 0px;
	/*border-color: hsl(210, 0%, 85%);*/
	background-color: hsl(210, 0%, 95%);
}

/*通用高亮按钮*/
/*Highlighted Push Button*/

QPushButton[isHighlighted=true],QToolButton[isHighlighted=true]
{
	color: #ffffff;
	border-style: solid;
	border-width: 1px;
	border-radius: 3px;
	border-color: hsl(210, 100%, 30%);
	background-color: hsl(210, 100%, 40%);
}
QPushButton[isHighlighted=true]:hover,QToolButton[isHighlighted=true]:hover
{
	border-color: hsl(210, 80%, 35%);
	background-color: hsl(210, 80%, 45%);
}
QPushButton[isHighlighted=true]:pressed,QToolButton[isHighlighted=true]:pressed
{
	border-color: hsl(210, 60%, 40%);
	background-color: hsl(210, 60%, 50%);
}
QPushButton[isHighlighted=true]:disabled,QToolButton[isHighlighted=true]:disabled
{
	color: hsl(210, 0%, 100%);
	border-width: 0px;
	/*border-color: hsl(210, 0%, 60%);*/
	background-color: hsl(210, 0%, 70%);
}

/*工具菜单按钮附件*/
/*Tool Button SP*/


/*通用列表控件*/
/*QListWidget*/

QListWidget {
    show-decoration-selected: 0;
}
QListWidget::item:hover {
    background-color: hsl(210, 15%, 95%);
}

/*通用行编辑框*/
/*Line Edit*/

QLineEdit
{
	color: hsl(0, 0%, 10%);
	selection-color: hsl(0, 0%, 100%);
	border-style: solid;
	border-width: 1px;
	border-radius: 3px;
	border-color: hsl(210, 5%, 90%);
	background-color: hsl(210, 5%, 100%);
	selection-background-color: hsl(210, 100%, 40%);
	padding: 0px 0px 0px 5px;
}
QLineEdit:hover
{
	color: hsl(0, 0%, 5%);
	border-color: hsl(210, 10%, 88%);
	background-color: hsl(210, 10%, 98%);
}
QLineEdit:focus
{
	border-bottom-color: hsl(210, 100%, 40%);
}
QLineEdit:disabled
{
	color: hsl(210, 0%, 60%);
	border-color: hsl(210, 0%, 85%);
	background-color: hsl(210, 0%, 95%);
}
QLineEdit:read-only
{
	border-color: hsl(210, 0%, 95%);
	background-color: hsl(210, 0%, 98%);
}

/*组合框及其附件*/
/*Combo box*/

QComboBox
{
	color: hsl(0, 0%, 10%);
	selection-color: hsl(0, 0%, 100%);
	border-style: solid;
	border-width: 1px;
	border-radius: 3px;
	border-color: hsl(210, 5%, 90%);
	background-color: hsl(210, 5%, 100%);
	selection-background-color: hsl(210, 100%, 40%);
	padding: 0px 0px 0px 5px;
}
QComboBox:hover
{
	color: hsl(0, 0%, 5%);
	border-color: hsl(210, 10%, 88%);
	background-color: hsl(210, 10%, 98%);
}
QComboBox:on
{
	border-color: hsl(210, 100%, 40%);
}
QComboBox:disabled
{
	color: hsl(210, 0%, 60%);
	border-color: hsl(210, 0%, 85%);
	background-color: hsl(210, 0%, 95%);
}
QComboBox::drop-down
{
	width: 0px;
	border: none;
}
QComboBox::down-arrow
{
	padding: 0px 32px 0px 0px;
	image: url(:/Resource/icons/drop-arrow-down);
}
QComboBox::down-arrow:on
{
	image: url(:/Resource/icons/drop-arrow-up);
}
QComboBox QAbstractItemView
{
	color: hsl(0, 0%, 10%);
	border-style: solid;
	border-width: 1px;
	border-radius: 3px;
	border-color: hsl(210, 100%, 40%);
	background-color: hsl(210, 5%, 100%);
	outline: 0px;
}
QComboBox QAbstractItemView::item
{
	color: hsl(0, 0%, 10%);
	border-style: none;
	background-color: hsl(210, 5%, 100%);
}
QComboBox QAbstractItemView::item:hover
{
	color: hsl(0, 0%, 5%);
	background-color: hsl(210, 10%, 98%);
}
QComboBox QAbstractItemView::item:selected
{
	color: #ffffff;
	background-color: hsl(210, 100%, 40%);
}

/*通用页面滑动条*/
/*Scroll Bar*/

QScrollBar
{
    background-color: hsl(210, 5%, 100%);
    padding: 12px 2px 12px 2px;
	width: 12px;
}
QScrollBar::handle
{
    background-color: hsl(210, 5%, 80%);
    border-radius: 2px; 
}
QScrollBar::handle:hover
{
    border-color: hsl(210, 10%, 78%);
}
QScrollBar::add-line:vertical
{
	height:12px;
	width:12px;
    border-image:url(:/Resource/icons/drop-arrow-down);
    subcontrol-position:bottom;
}
QScrollBar::sub-line:vertical
{
    height:12px;
	width:12px;
    image:url(:/Resource/icons/drop-arrow-up);
    subcontrol-position:top;
}

/*通用菜单内容*/
/*Menu &amp; Menu Content*/

QMenu
{
	color: hsl(0, 0%, 10%);
	border-style: solid;
	border-width: 1px;
	border-radius: 3px;
	border-color: hsl(210, 5%, 90%);
	background-color: hsl(210, 5%, 100%);
	padding: 5px 5px 5px 5px;
}
QMenu::item
{
	color: hsl(0, 0%, 10%);
	border-style: none;
	border-radius: 3px;
	background-color: hsl(210, 5%, 100%);
	padding: 4px 4px 4px 4px;
}
QMenu::item:selected
{
	color: hsl(0, 0%, 5%);
	background-color: hsl(210, 10%, 95%);
}
/* QMenu::item:selected
{
	color: #ffffff;
	background-color: hsl(210, 100%, 40%);
} */
QMenu::item:disabled
{
	color: hsl(210, 0%, 60%);
}
QMenu::separator
{
	height: 1px;
	background-color: hsl(210, 5%, 80%);
}

/*w_mainMenu下的特殊控件样式*/
/*special style under &quot;w_mainMenu&quot;*/

QWidget#w_mainMenu
{
	background-color: hsl(210, 60%, 95%);
}
QWidget#w_mainMenu&gt;QPushButton
{
	color: hsl(0, 0%, 10%);
	font: 10pt &quot;微软雅黑&quot;;
	border-style: solid;
	border-width: 1px, 0px, 0px, 0px;
	border-radius: 3px;
	border-color: hsl(210, 5%, 90%);
	border-right-color: hsl(210, 5%, 100%);
	border-left-color: hsl(210, 5%, 100%);
	background-color: hsl(210, 5%, 100%);
}
QWidget#w_mainMenu&gt;QPushButton:hover
{
	color: hsl(0, 0%, 5%);
	background-color: hsl(210, 10%, 98%);
}
QWidget#w_mainMenu&gt;QPushButton:pressed
{
	color: hsl(0, 0%, 0%);
	background-color: hsl(210, 15%, 95%);
}
QWidget#w_mainMenu&gt;QPushButton:checked
{
	color: #ffffff;
	background-color: hsl(210, 100%, 40%);
}
QWidget#w_mainMenu&gt;QPushButton:checked:hover
{
	color: #ffffff;
	background-color: hsl(210, 80%, 45%);
}
QWidget#w_mainMenu&gt;QPushButton:checked:pressed
{
	color: #ffffff;
	background-color: hsl(210, 60%, 50%);
}
QWidget#w_mainMenu&gt;QPushButton:disabled
{
	color: hsl(210, 0%, 60%);
	border-width: 0px;
	/*border-color: hsl(210, 0%, 85%);*/
	background-color: hsl(210, 0%, 95%);
}
QWidget#w_mainMenu&gt;QLabel#lb_progStatus
{
	color: #ffffff;
	background-color: hsl(0, 0%, 40%);
}
QWidget#w_mainMenu&gt;QLabel#lb_progStatus[status=&quot;ready&quot;]
{
	background-color: hsl(210, 100%, 40%);
}
QWidget#w_mainMenu&gt;QLabel#lb_progStatus[status=&quot;working&quot;]
{
	background-color: hsl(200, 100%, 40%);
}
QWidget#w_mainMenu&gt;QLabel#lb_progStatus[status=&quot;warning&quot;]
{
	background-color: hsl(50, 100%, 40%);
}
QWidget#w_mainMenu&gt;QLabel#lb_progStatus[status=&quot;fatal&quot;]
{
	background-color: hsl(0, 100%, 40%);
}

/*OnlineSearch&gt;SearchResult下的特殊控件样式*/
/*special style under &quot;OnlineSearch&gt;SearchResult&quot;*/

QWidget#w_SearchResultU
{
	border-color: hsl(210, 5%, 90%);
	border-style: solid;
	border-width: 1px;
	border-radius: 3px;
}
QWidget#w_SearchResultU&gt;QWidget
{
	background-color: transparent;
}
QWidget#w_SearchResultU&gt;QWidget#w_title&gt;QLabel#lb_titleText
{
	font: 700 11pt &quot;Microsoft YaHei UI&quot;;
	background-color: transparent;
}

/*自定义的DialogBox特殊样式*/
/*special style under &quot;DialogBoxU&quot;*/

QDialog#DialogBoxU
{
	background-color: hsl(210, 50%, 98%);
	font: 12pt bold &quot;Microsoft YaHei UI&quot;;
}
QDialog#DialogBoxU&gt;QWidget#title
{
	background-color: hsl(210, 50%, 95%);
}
QDialog#DialogBoxU&gt;QWidget#title QLabel
{
	background-color: hsl(210, 50%, 95%);
	font: 700 16pt &quot;Microsoft YaHei UI&quot;;
}
QDialog#DialogBoxU&gt;QWidget#title&gt;QLabel#title_icon
{
	padding: 0px 2px 0px 5px;
}
QDialog#DialogBoxU&gt;QWidget#title&gt;QLabel#title_icon[icon=info_blue]
{
	image: url(:/icons/info_blue);
}
QDialog#DialogBoxU&gt;QWidget#title&gt;QLabel#title_icon[icon=warning_yellow]
{
	image: url(:/icons/warning_yellow);
}
QDialog#DialogBoxU&gt;QWidget#title&gt;QLabel#title_icon[icon=warning_red]
{
	image: url(:/icons/warning_red);
}
QDialog#DialogBoxU&gt;QWidget#title&gt;QLabel#title_icon[icon=error_red]
{
	image: url(:/icons/error_red);
}
QDialog#DialogBoxU&gt;QWidget#title&gt;QLabel#title_icon[icon=settings_blue]
{
	image: url(:/icons/settings_blue);
}
QDialog#DialogBoxU&gt;QWidget#title&gt;QLabel#title_icon[icon=critical]
{
	image: url(:/icons/critical);
}
QDialog#DialogBoxU&gt;QLabel#content
{
	font: 12pt &quot;Microsoft YaHei UI&quot;;
	padding: 10px 10px 10px 10px;
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="w_windowControl" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: hsl(210, 50%, 95%);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="HS_windowControl">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lb_dataCollectingTag">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 2px 10px 2px 10px;
color: hsl(210, 0%, 100%);
background-color: hsl(210, 100%, 40%);
border-radius: 4px;</string>
        </property>
        <property name="text">
         <string>Now Collecting Data</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_closeWindow">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:transparent;
font: 12pt &quot;Microsoft YaHei UI&quot;;
border-color:transparent;
padding: 2px 10px 2px 10px;
color: hsl(210, 0%, 50%);</string>
        </property>
        <property name="text">
         <string>✕</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="w_header" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="lb_coverImg">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>110</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>110</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 12pt &quot;Microsoft YaHei UI&quot;;
color: hsl(210, 15%, 90%);
background-color: hsl(210, 15%, 97%);

border-style: solid;
border-width: 2px;
border-radius: 5px;
border-color: hsl(210, 0%, 90%);</string>
        </property>
        <property name="text">
         <string>Loading
Image</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="w_briefInfo" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lb_titleText">
           <property name="styleSheet">
            <string notr="true">font: 700 14pt &quot;Microsoft YaHei UI&quot;;</string>
           </property>
           <property name="text">
            <string>Music Title There</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lb_subTitle">
           <property name="styleSheet">
            <string notr="true">font: 700 10pt &quot;Microsoft YaHei UI&quot;;
color: hsl(210, 10%, 50%);</string>
           </property>
           <property name="text">
            <string>Trans-title or Sub-title Here</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lb_artists">
           <property name="text">
            <string>Artists: Artist1, Artist2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lb_album">
           <property name="text">
            <string>Ablum: Ablum Name Here</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="VS_headerPusher">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="w_controller" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QPushButton" name="btn_download">
           <property name="text">
            <string>Download</string>
           </property>
           <property name="isHighlighted" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_play">
           <property name="text">
            <string>Play</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="VS_controllerPusher">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="w_bySourceZone" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QListWidget" name="lw_sourceChoose">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Music Source 1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Music Source 2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Music Source 3</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="w_bySourceInfoZone" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
